modules/tga/image_loader_tga.cpp

```diff
@@ -148,9 +148,9 @@ Error ImageLoaderTGA::convert_to_image(Ref<Image> p_image, const uint8_t *p_buff
 					uint8_t a = 0xff;
 
 					if (p_header.color_map_depth == 24) {
-						r = (p_palette[(index * 3) + 0]);
```

Comment:
Might be worth adding a comment to explain why it's in this order.

Reply:
Added, according to http://www.paulbourke.net/dataformats/tga/